src/Command/InitializeAdherentEmailSubscriptionHistoryCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\AdherentEmailSubscriptionHistory;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Internal\Hydration\IterableResult;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class InitializeAdherentEmailSubscriptionHistoryCommand extends ContainerAwareCommand
+{
+    private const COMMAND_NAME_ALIAS = 'app:adherent:iesh';
+    private const BATCH_SIZE = 50;
+
+    protected static $defaultName = 'app:adherent:initialize-email-subscriptions-history';
```

Comment:
is this property useful ?

Reply:
Yes it's one way to enable lazy commands in Symfony

Reply:
oh yes it is \o/

Reply:
https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading